### PR TITLE
Additional docs

### PR DESCRIPTION
**Pull request by apdibbo**
Additional docs in the run up to giving a demo of Octavia
- docker nginx tutorial
- octavia quick start guide
- update on blocking automatic updates safely

### CODE REVIEW

File: source/Octavia/QuickStart.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+.. code-block:: bash
+
+  ssh username@130.246.215.87 -p 200
+
+Viewing details
+---------------
+
+If you click on the Name of the load balancer you can view details of the listerners and pools
+
+Unfortunately a number of the "Operating Status" fields always report "Offline" even when the
```
Comment (DavidFair) [inline source/Octavia/QuickStart.rst]: `they are online`

File: source/Octavia/QuickStart.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+You can add another Listener to this load balancer by hitting "Create Listener" and following similar steps to the above
```
Comment (DavidFair) [inline source/Octavia/QuickStart.rst]: Could we create a link, above the section you want to link to add
`.. _my_link_name:`

then to create the link
```
:ref:`my_link_name`
```

File: source/howto/PreventAutomaticUpdates.rst [hunk 3]
```diff
@@ -2,7 +2,21 @@
...
+
+.. code-block:: bash
+
+  sudo chown your-username /etc/yum/pluginconf.d/versionlock.list
+
+Then add any packages to the above file in the same format as the other entries.
+
+Preventing Automatic Updates completely
+---------------------------------------
+This is not recommended must only be done with the express written consent of the Cloud team. This can be obtained by emailing cloud-support@gridpp.rl.ac.uk
```
Comment (DavidFair) [inline source/howto/PreventAutomaticUpdates.rst]: `and must only`

**Review by DavidFair — CHANGES_REQUESTED** [review]
Some minor typos, sorry about the delay just seen this PR

**Comment by apdibbo** [comment]
I have addressed your comments

File: source/Octavia/QuickStart.rst [hunk 4]
```diff
@@ -25,6 +25,8 @@ On the "Load Balancer Details" screen enter a name for the load balancer and sel
 
 .. image:: /assets/howtos/CreateALoadBalancer/3-create-load-balancer-details.png
 
+.. _create_octavia_listener
```
Comment (DavidFair) [inline source/Octavia/QuickStart.rst]: Your missing a `:`

`.. _create_octavia_listener:`

**Review by DavidFair — CHANGES_REQUESTED** [review]
One small typo, otherwise looks good to merge - thanks

**Comment by apdibbo** [comment]
Typo fixed

**Review by DavidFair — APPROVED** [review]

**Pull request merged**